src/index.js

```diff
@@ -0,0 +1,39 @@
+import { MongoClient } from 'mongodb';
+
+const uri = 'mongodb://localhost:27017';
```

Comment:
Consider using an actual port number for localhost that is ```mongodb://127.0.0.1:2707``` to avoid DNS lookup and ipv6 usage.